### ISSUE TITLE
Include chest and other container drops in BlockBreakEvent

### ISSUE THREAD

**Issue by dktapps** [pmmp/PocketMine-MP#4451]
## Description
Items dropped by containers due to players destroying blocks should be included by BlockBreakEvent.
Right now it's difficult to prevent containers from dropping items. This is especially annoying in creative servers where you want to prevent creative players from dropping items, since they can just put the items into a chest and then break the chest instead of dropping it normally and getting it cancelled.

Right now, there is no way to reliably detect when an item drop was caused by destroying a container such as a chest.

## Alternative methods
You can try and hack around with ItemSpawnEvent? not ideal ...

**Comment by nathanswanson** [comment]
Isn't it possible to cancel the block break event if a block is a container? With that you can grab the chest tile and do what you want with the inventory.

**Comment by dktapps** [comment]
That's true, I hadn't considered that. Still, it's an extra step that shouldn't really be necessary.
